docs/DeveloperGuide.md

```diff
@@ -552,21 +552,99 @@ testers are expected to do more *exploratory* testing.
 
 1. _{ more test cases …​ }_
 
-### Deleting a person
+### Command Testing
+#### Adding A Person or A Company
+1. Adding a person
 
-1. Deleting a person while all persons are being shown
+    1. Test case: `add p n/Tom p/12345678 e/tom@gmail.com a/Kent Ridge Road t/friend`<br>
+       Expected: A person contact with the given information is added to the end of the person contact list. 
+            Details of the newly added person contact shown in the status message. Timestamp in the status bar is updated.
```

Comment:
I'm not quite sure what the timestamp in the status bar is. Do you have a screenshot of it?

Reply:
I refer to the AB3 UG. It indicates the message shown in the status bar I think.